config/zones/MX-BC.yaml

```diff
@@ -11,6 +11,17 @@ fallbackZoneMixes:
       gas: 0.923
       wind: 0.077
   source: "Electricity Maps & Dra. Gabriela Mu\xF1oz Mel\xE9ndez"
+emissionFactors:
+  direct:
+    unknown:
+      - datetime: '2017-01-01'
+        source: "Electricity Maps & Dra. Gabriela Mu\xF1oz Mel\xE9ndez"
```

Comment:
I think we can get rid of the unicode encoding here, make it looks a little better when looking at the file.

Reply:
Not entierly sure if our whole pipeline will support it, so to be sure I would rather keep it as it is :)

Reply:
Yaml it self supports it but that is a valid point.

let's keep it

Reply:
Though I hope it supports it but I don't want to spend time fixing it if it doesn't right now 😅